stories/components.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-env node */
+import hbs from 'htmlbars-inline-precompile';
+import { storiesOf } from '@storybook/ember';
+import moment from 'moment';
+import crossfilter from 'crossfilter';
+import content from './content.json';
+import { action } from '@storybook/addon-actions';
+
+const initializeStories = () => {
+    storiesOf('Components', module)
+        .add('Column Chart', () => {
+            return {
+                template: hbs`
+                {{column-chart 
+                    dimension=dimension
+                    group=group
+                    type=type
+                    series=series
+                    colors=colors
+                    height=height
+                    xAxis=xAxis
+                    yAxis=yAxis
+                    click=(action onClick)
+                }}`,
+                context: {
+                    dimension: dimensions,
+                    group: groups,
+                    type: 'GROUPED',
+                    series: [{ title: 'Offered Calls', hatch: false }],
+                    colors,
+                    height: 200,
+                    xAxis: {
+                        domain: [moment('10/31/2016'), moment('12/03/2016')],
```

Comment:
changing this to `[moment('2016-10-31T00:00:00.000Z'), moment('2016-12-03T00:00:00.000Z')]` will get rid of the `Deprecation warning: value provided is not in a recognized RFC2822 or ISO format` message